### ISSUE TITLE
Question about Table 1 and the joint_det setting

### ISSUE THREAD

**Issue by WeitaiKang** [yanmin-wu/EDA#19]
Dear authors,

Thanks for your great job. I have question about the --joint_det setting. I find it is set in all the scripts, which means you always train the model with additional utterance data constructed from ScanNet object labels.

Is it a widely-used protocol in 3D Visual Grounding? And is the performance in Tab. 1 come from the joint_det setting?

Best,

**Comment by WeitaiKang** [comment]
I have check the train log your provide:

[07/27 05:51:01 bdetr]: ... 'joint_det': True ...
[07/27 05:59:39 bdetr]: length of training dataset: 48655
[07/27 05:59:39 bdetr]: length of testing dataset: 9508

As shown above which indicates the size of data in your train log, i find that the train set is larger than scanrefer, and it is the amount of --joint_det. Also the configure set joint_det as True. So your performance (Table 1) comes from training on both scanrefer and scannet. Is it that a widely-used setting? It seems no mention in your paper *4.1.1 Experiment settings*

**Comment by yanmin-wu** [comment]
Thank you for your attention. 
We keep our settings consistent with [BUTD-DETR](https://github.com/nickgkan/butd_detr). This strategy is fair for the following reasons:
(1) In ScanRefer, it is necessary to first obtain the object boxes and labels in ScanNet to further process and obtain box-description pairs. Therefore, we do not use more training data than other 3DVG methods.
(2) In other two-stage methods, it is necessary to first use ScanNet detection data to pre-train the detector, and then train the second-stage VG model. We train the detection task and the VG task simultaneously in the same model.

**Comment by WeitaiKang** [comment]
Thanks for your clarification! You mention that previous methods first pretrain ScanNet detection data, then train VG data. In contrast, your method (and BUTD-DETR) join train this two kinds of data, to keep it a fair training cost.

However, your method (and BUTD-DETR) do have already pretrained on ScanNet detection data (I mean the PointNet++ in your model). Does it mean that you train ScanNet detection data twice (once in pretrain PointNet++, once in join train VG)? Or I miss something here. Is it a fair setting from the training cost perspective, compared with previous methods (two-stage methods)?

Look forward to your response!

**Comment by yanmin-wu** [comment]
Thank you for pointing that out. 
Yes, by default, we use the pretrained pointnet++ checkpoint provided by BUTD-DETR. However, according to our experiments, training pointnet++ from scratch along with our model, without using the pretrained model, can also achieve comparable performance. 
[Similar issue](https://github.com/yanmin-wu/EDA/issues/8#issuecomment-1537683989).

**Comment by WeitaiKang** [comment]
Thanks for your reply! It does answer my question.
